### PR TITLE
added platformCreation script notice and fixed get parameters in time…

### PR DESCRIPTION
**Pull request by mbogor**
#### What's this PR do?
- adds a message alerting the graduation slack channel when the grad script is running
- fixed whenIwork.get parameters call in timeOff requests 
- calls only for "workflow_state": "available" courses' grade logs; significantly reduces the amount of api calls necessary to make
#### What are the relevant tickets?
INT - 291
#### Questions:

…off requests

### CODE REVIEW

File: jobs/scheduling/TimeOffRequests.js [hunk 1]
```diff
@@ -49,8 +49,8 @@ function filterRequestsAndHandleShifts (allRequests) {
 
 function retrieveOverlappingShifts (request) {
   var shiftSearchParams = createShiftSearchParams(request);
-
-  WhenIWork.get('shifts?include_objects=false', shiftSearchParams, function(response) {
+  shiftSearchParams.include_objects=false;
```
Comment (jrauschenberg) [inline jobs/scheduling/TimeOffRequests.js]: I wonder if you could combine this function with createTimeOffSearchParams. I know it has extra params, but I think WiW will ignore them if they don't apply and/or aren't filled in. Worth thinking about.

**Comment by jrauschenberg** [comment]
One comment. Otherwise good to go. 👍

File: jobs/scheduling/TimeOffRequests.js (hunk 1 above)
Comment (mbogor) [inline jobs/scheduling/TimeOffRequests.js]: i like that, fixed it.

**Comment by tongxiang** [comment]
👍

**Pull request merged**